cluster/events.go

```diff
@@ -0,0 +1,20 @@
+package cluster
+
+import "github.com/docker/docker/api/types/events"
+
+// Event is exported
+type Event struct {
+	events.Message
+	Engine *Engine `json:"-"`
+}
+
+// EventHandler is exported
+type EventHandler interface {
+	Handle(*Event) error
```

Comment:
It looks like at this point all implementations of this method always return `nil` instead of an error. How about making this not return anything?